starter_templates/main.rs

```diff
@@ -0,0 +1,11 @@
+
+//main objective is to start TCP server on port 6379 , which is the default port that Redis uses
```

Comment:
Please use comments that are similar to [the other languages](https://github.com/codecrafters-io/languages/blob/master/starter_templates/redis/python/app/main.py). There's really only one comment needed - "Uncomment this to pass the first stage".

Reply:
acknowledged sir